config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -66,8 +66,9 @@ periodics:
       - --env=CL2_LOAD_TEST_THROUGHPUT=50
       - --env=KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS=--profiling --kube-api-qps=200 --kube-api-burst=200
       - --env=KUBEMARK_SCHEDULER_TEST_ARGS=--profiling --kube-api-qps=200 --kube-api-burst=200
+      - --env=HEAPSTER_MACHINE_TYPE=e2-standard-8
```

Comment:
If it has the same size as the other machines, it isn't needed, right?

Reply:
Makes sense. OTOH, we include `preset-e2e-kubemark-gce-scale` preset here where we have such an entry:
```
- name: HEAPSTER_MACHINE_TYPE
    value: "n1-standard-8"
```
hence my addition here. Shall we get rid of this entry in that preset altogether then?

Reply:
Let's change the preset then (I know it will affect other jobs, but that's probably fine).

Reply:
Done.